src/test/datascience/notebook/interrupRestart.vscode.test.ts

```diff
@@ -73,8 +74,14 @@ suite('DataScience - VSCode Notebook - Restart/Interrupt/Cancel/Errors (slow)',
         vscEditor = vscodeNotebook.activeNotebookEditor!;
         // Disable the prompt (when attempting to restart kernel).
         dsSettings.askForKernelRestart = false;
+        traceInfo(`Start Test (completed) ${this.currentTest?.title}`);
+    });
+    teardown(async function () {
+        traceInfo(`End Test ${this.currentTest?.title}`);
+        await closeNotebooks(disposables);
+        await closeNotebooksAndCleanUpAfterTests(disposables.concat(suiteDisposables));
```

Comment:
This function will ensure we kill all INotebook objects between each test & wait for it to complete.
This wasn't interfering with tests, but output form this was causing confusions when trying to diagnose test failures.